src/orderbook/depth_finite.jl

```diff
@@ -29,14 +29,14 @@ function OrderBook(asks::Vector, bids::Vector)
 end
 
 
-function _price(v_level::Vector{Level{Tprice, Tvolume}}, volume::Volume; raise=false) where {Tprice, Tvolume}
-    @assert volume >= Volume(0) "volume must be positive or zero"
+function _price(v_level::Vector{Level{Tprice, Tvolume}}, volume::Tvolume; raise=false) where {Tprice, Tvolume}
+    @assert volume >= Tvolume(0) "volume must be positive or zero"
```

Comment:
Not sure if I should use

    Tvolume(0)

or (like @roshii mentioned previously)

    zero(Tvolume)

Reply:
I guess it is just a matter of code style

```
julia> @btime zero(Float64)
  0.024 ns (0 allocations: 0 bytes)

julia> @btime Float64(0)
  0.024 ns (0 allocations: 0 bytes)
```